spec.json

```diff
@@ -1,289 +1,314 @@
-{
-  "nio/MSSQLDelete": {
-    "version": "0.2.0",
-    "description": "Creates a cursor object for an incoming signal and executes one command per signal.",
-    "categories": [
-      "Database"
-    ],
-    "properties": {
-      "command": {
```

Comment:
i think there is a way to derive this stuff from python reflection. See https://github.com/nio-blocks/communication/blob/master/spec.json